index.html

```diff
@@ -79,6 +79,212 @@ <h1 class="h1">I'm David. Glad to see you!</h1>
+            <p class="medium light-grey">Back End Dev</p>
+            <span></span>
+            <p class="medium light-grey">2015</p>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="project-language">
+            <li class="medium dark-purple">HTML</li>
+            <li class="medium dark-purple">CSS</li>
+            <li class="medium dark-purple">JS</li>
+          </ul>
+          <button class="button-outlined medium">See project</button>
+        </div>
+        <div class="project-item">
+          <img
+            src="images/project4_snapshot.png"
+            alt="project1"
+            width="295"
+            height="220"
+          />
+          <h3 class="h1 bold">Multi-Post Stories</h3>
+          <div class="captions">
+            <p class="bold">CANOPY</p>
+            <span></span>
+            <p class="medium light-grey">Back End Dev</p>
+            <span></span>
+            <p class="medium light-grey">2015</p>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="project-language">
+            <li class="medium dark-purple">HTML</li>
+            <li class="medium dark-purple">CSS</li>
+            <li class="medium dark-purple">JS</li>
+          </ul>
+          <button class="button-outlined medium">See project</button>
+        </div>
+      </section>
+
+      <section class="about-me">
+        <div class="description">
+          <h3 class="h1 bold">About Myself</h3>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website. Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <p class="medium purple">LET'S CONNECT</p>
+          <ul class="icons">
+            <li>
+              <a href="https://twitter.com/DAndrianarivo"
+                ><i class="fa-brands fa-twitter fa-lg"></i
+              ></a>
+            </li>
+            <li>
+              <a
+                href="https://www.linkedin.com/in/david-andrianarivo-3692101b6/"
+                ><i class="fa-brands fa-linkedin fa-lg"></i
+              ></a>
+            </li>
+            <li>
+              <a href="https://medium.com/@andrianarivo.david"
+                ><i class="fa-brands fa-medium fa-lg"></i
+              ></a>
+            </li>
+            <li>
+              <a href="https://github.com/andrianarivo"
+                ><i class="fa-brands fa-github fa-lg"></i
+              ></a>
+            </li>
+          </ul>
+          <form method="get" action="./resume.pdf">
+            <button class="button-outlined medium">Get my resume</button>
+          </form>

```

Comment:
- [ ] For semantic reasons, there's no need to use the form here on get resume button. the form element is only used where users are required to submit some infos. Kindly consider removing that.

`Hint: You may consider using the button as you did or an anchor link and set the download attribute` 
example
```html
<a href="/images/myw3schoolsimage.jpg" download>
```
You can read more [here](https://www.w3schools.com/tags/att_a_download.asp)